index.js

```diff
@@ -42,4 +42,16 @@ module.exports = function() {
     })
   });
 
-}
\ No newline at end of file
+  const load_listener = (log) => {
+    const result = log.match(/]:\sDone/);
+    // server launch done
+    if(result) {
+      // send load event
+      server.emit('load');
+      // end listen
+      server.removeListener('console', load_listener);
+    }
+  }
+  server.on('console', load_listener);
```

Comment:
Could we make this more syntactically consistent with the rest of the listeners? 

Does `load` feel a little vague? maybe `start`? I'm not sure..

Also add a timestamp to the emit